### PR TITLE
motd: Add tracker URL and experimental warning

### PR DESCRIPTION
**Pull request by jlebon**
Make it more obvious to anyone using these artifacts that this is a
rough sketch of how FCOS might look and that things can change. Also add
a link back to the tracker to complement this.

### CODE REVIEW

File: fedora-coreos-base.yaml [hunk 1]
```diff
@@ -112,10 +112,15 @@ postprocess:
     # be below that.
     cat > /etc/motd <<EOF
     Fedora CoreOS (preview)
+    https://github.com/coreos/fedora-coreos-tracker
+    WARNING: All aspects subject to change, highly experimental
```
Comment (jlebon) [inline fedora-coreos-base.yaml]: Open to rewording this!

File: fedora-coreos-base.yaml [hunk 2]
```diff
@@ -112,10 +112,15 @@ postprocess:
...
     ---
 
     EOF
 
+    # And mark as preview in the os-release too, this then shows up in eg. GRUB2
+    sed -i -e 's/CoreOS/CoreOS preview/' $(realpath /etc/os-release)
```
Comment (dustymabe) [inline fedora-coreos-base.yaml]: want to use parentheses like we did in the motd?

**Comment by dustymabe** [comment]
LGTM, one nit

File: fedora-coreos-base.yaml (hunk 2 above)
Comment (dustymabe) [reply fedora-coreos-base.yaml]: I see :+1:

**Review by dustymabe — APPROVED** [review]

File: fedora-coreos-base.yaml [hunk 3]
```diff
@@ -112,10 +112,15 @@ postprocess:
     # be below that.
     cat > /etc/motd <<EOF
     Fedora CoreOS (preview)
+    https://github.com/coreos/fedora-coreos-tracker
```
[the remark this answers is not in the document]
Comment (dustymabe) [reply fedora-coreos-base.yaml]: my $.02: I like hitting the top level readme because the first paragraph gives some background on what the repo is for.

**Comment by dustymabe** [comment]
i be mergin

**Pull request merged**